cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
at the moment the `Add` command has a `--instance-only` option, but I always define `setup-users` as for all instances. it is moves to add, I can allow people to decide, but default is 'all instances'.